command/aarelayer/params.go

```diff
@@ -54,6 +60,28 @@ func (rp *aarelayerParams) validateFlags() error {
 	return sidechainHelper.ValidateSecretFlags(rp.accountDir, rp.configPath)
 }
 
+func (rp *aarelayerParams) getLogger() (hclog.Logger, error) {
+	if rp.logPath != "" {
+		logFileWriter, err := os.Create(rp.logPath)
+		if err != nil {
+			return nil, fmt.Errorf("could not create log file, %w", err)
+		}
+
+		return hclog.New(&hclog.LoggerOptions{
+			Name:       "aarelayer",
+			Output:     logFileWriter,
+			Level:      hclog.LevelFromString(rp.logLevel),
+			JSONFormat: false,
+		}), nil
+	}
+
+	return hclog.New(&hclog.LoggerOptions{
```

Comment:
You could remove the nested return and just set a local variable `var outputter io.Writer` and set it in the lower return. In case the log path was not provided, it will be `nil`, and `hclog` will use its `DefaultOutput`.